fdbserver/Ratekeeper.actor.cpp

```diff
@@ -322,9 +322,22 @@ class RatekeeperImpl {
 						p.lastThrottledTagChangeId = self.tagThrottler->getThrottledTagChangeId();
 						p.lastTagPushTime = now();
 
-						reply.throttledTags = self.tagThrottler->getClientRates();
-						bool returningTagsToProxy =
-						    reply.throttledTags.present() && reply.throttledTags.get().size() > 0;
+						bool returningTagsToProxy{ false };
+						if (SERVER_KNOBS->ENFORCE_TAG_THROTTLING_ON_PROXIES) {
+							reply.proxyThrottledTags = self.tagThrottler->getRates();
+							if (reply.proxyThrottledTags.present()) {
```

Comment:
I think this is always true because the return value of `self.tagThrottler->getRates()` is not an `Optional<>`